src/app/layouts/global/Footer.tsx

```diff
@@ -1,11 +1,315 @@
+
+      <section className="flex flex-col items-center">
+        <MainLogo />
+        <fieldset className="flex h-max gap-4">
+          <legend className="pb-5">
+            <strong className="text-primary-700">Стеж за нами:</strong>
+          </legend>
+          <a href="/"><TweeterLogo /></a>
+          <a href="/"><FacebookLogo /></a>
+          <a href="/"><TikTokLogo /></a>
+        </fieldset>
+      </section>
+
+      <section className="grid items-start justify-center justify-items-center gap-6 pb-5">
+        <fieldset className="flex w-full flex-col gap-4">
+          <legend className="pb-5  text-primary-700">
+            <strong>Зв’язатися з нами:</strong>
+          </legend>
+          <address>
+            <a href="/" className="flex space-x-2">
+              <WhatsAppLogo />
+              <span className="text-primary not-italic">+380991235678</span>
+            </a>
+          </address>
+          <address>
+            <a href="/" className="flex space-x-2">
+              <EmailLogo />
+              <span className="text-primary not-italic">gifthub@gmail.com</span>
+            </a>
+          </address>
+          <address>
+            <a href="/" className="flex space-x-2">
+              <LocationPointLogo />
+              <span className="text-primary not-italic">м.Київ; вулиця Олеся Гончара</span>
+            </a>
+          </address>
+        </fieldset>
+
+        <fieldset className="flex w-full flex-col">
+          <legend className="pb-5  text-primary-700">
+            <strong>Ми на зв’язку у месенджерах:</strong>
+          </legend>
+          <ul className="flex space-x-4">
+            <li>
+              <a href="/"><FacebookLogo /></a>
+            </li>
+            <li>
+              <a href="/"><TelegramLogo /></a>
+            </li>
+          </ul>
+        </fieldset>
+      </section>
+
+      <section className="flex justify-center">
+        <fieldset className="flex flex-col items-start">
+          <legend className="pb-5  text-primary-700">
+            <strong>Допомога</strong>
+          </legend>
+          <ul className="flex flex-col gap-3">
+            <li>
+              <a href="/" className="font-light text-primary">Часті питання</a>
+            </li>
+            <li>
+              <a href="/" className="font-light text-primary">Договір оферти</a>
+            </li>
+            <li>
+              <a href="/" className="font-light text-primary">Оплата і доставка</a>
+            </li>
+            <li>
+              <a href="/" className="font-light text-primary">Умови повернення</a>
+            </li>
+            <li>
+              <a href="/" className="font-light text-primary">Політика конфіденційності</a>
+            </li>
+          </ul>
+        </fieldset>
+      </section>
+
+      <section className="col-span-full flex items-center justify-between border-t px-11" style={{ borderLeft: "0px" }}>
```

Comment:
why do you use inline style?

Reply:
Here i tried to use this class:
https://tailwindcss.com/docs/divide-width
But i do not found how to avoid extra line for my last grid block.
So, just use inline style. If u have better way to fix this, let's do it.